src/UI/components/home/statsBox/statsBox.test.tsx

```diff
@@ -12,10 +12,8 @@ describe('StatsBox component', () => {
   };
   it('renders the currently hardcoded stats', () => {
     render(<StatsBox />, state);
-    expect(screen.getByText('Downloads')).toBeInTheDocument();
-    expect(screen.getByText('Data Points')).toBeInTheDocument();
-    expect(screen.getByText('Countries')).toBeInTheDocument();
-    expect(screen.getByText('Citations')).toBeInTheDocument();
-    expect(screen.getByText('Species')).toBeInTheDocument();
+    expect(screen.getByText('vector records')).toBeInTheDocument();
+    expect(screen.getByText('visited...')).toBeInTheDocument();
+    expect(screen.getByText('by..')).toBeInTheDocument();
```

Comment:
It'd be nice to have a test for the initial action dispatch, in case someone removes it.